src/main.js

```diff
@@ -227,6 +228,22 @@ function showPosterGrid() {
   }
 }
 
+function deletePoster(event) {
```

Comment:
Also want to note that I'm pretty sure this is called event bubbling. This is definitely a topic we will both need to go over a couple of times before we turn in the completed project.